sections/findify-autocomplete.liquid

```diff
@@ -0,0 +1,101 @@
+{%- assign page_url = content_for_header | split:'"pageurl":"' | last | split:'"' | first | split: request.host | last | replace:'\/','/' | replace:'%20',' ' | replace:'\u0026','&'  -%}
+
+{%- assign query = page_url | split: 'findify-autocomplete&q=' -%}
+{%- assign suggestionArray = query[1] | split: '&suggestion=' | first | split: 'isMobile' | first -%}
+{%- assign suggestions = suggestionArray | split: ',' -%}
+{%- assign productHandlers = query[1] | split: '&suggestion=' %}
+{%- assign isMobile = query[1] | split: '&isMobile=' | last -%}
+{%- assign array = productHandlers[0] | split: ',' %}
+{%- assign suggestionsTemplate = "vertical"-%}
+{%- assign viewAllPosition = "bottom" -%}
+{%- assign findifyAutocompleteContentTitle = settings.findifyAutocompleteContentTitle -%}
+{%- assign findifyAutocompleteContentID = settings.findifyAutocompleteContentID -%}
+{%- assign class = "findify-autocomplete findify-autocomplete-fullscreen" -%}
+{%- assign productCard = '' | append: "findify-product-card" -%}
+
+<div id="findify-autocomplete" class="findify-autocomplete findify-autocomplete-fullscreen hidden">
+<div class="findify-autocomplete-content{% if findifyAutocompleteContentTitle.size > 0 and findifyAutocompleteContentID.size > 0 %}columns-3{% endif %}">
```

Comment:
Here can be simpliefied and cleaned up

no comuns-3 needed.

We can simply make 3 columns with display: flex
And on the inner divs
flex: 1 (suggestions)
flex: 2 (products) or even 3
flex: 1 (when we will add contents)
OR
keep products display: flex and the other fixed width.